client/engine/engine.go

```diff
@@ -453,6 +521,24 @@ func (e *Engine) attemptProgress(objective protocols.Objective) (outgoing Object
 	return
 }
 
+func (e Engine) registerVirtualChannelWithManagers(vfo virtualfund.Objective) error {
+	prefund := vfo.V.PreFundState()
```

Comment:
Nit: would the post fund make more sense? They should have the same outcome, of course...

Reply:
updated in fb08bac